packages/medusa-payment-stripe/src/helpers/stripe-base.js

```diff
@@ -59,12 +59,22 @@ class StripeBase extends AbstractPaymentService {
   }
 
   getPaymentIntentOptions() {
-    return {
-      setup_future_usage:
-        this?.paymentIntentOptions?.setup_future_usage ?? "on_session",
-      payment_method_types: this?.paymentIntentOptions
-        ?.payment_method_types || ["cart"],
+    const options = {}
```

Comment:
suggestion:

```ts
Object.entries(this. paymentIntentOptions ?? {}).forEach(([key, value]) => options[key] = value)
```

Reply:
it's probably better to have them explicity here.

Reply:
I don’t have strong opinion as it will be typed when we will migrate it

Reply:
Yeah, I'll keep them explicitly for now. And then when we migrate to TS, we can revisit the implementation.